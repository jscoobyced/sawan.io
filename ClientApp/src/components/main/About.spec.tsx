import { shallow } from 'enzyme';
import React from 'react';
import { About } from './About';

test('About component is unchanged.', () => {
  const about = shallow(<About />);
  expect(about.find('div')).toHaveLength(1);
});
