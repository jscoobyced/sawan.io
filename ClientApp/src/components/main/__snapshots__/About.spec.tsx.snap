// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component is unchanged. 1`] = `
<div>
  About
</div>
`;
